### PR TITLE
Notify admins about assigned Virtual Card

### PR DESCRIPTION
**Pull request by kewitz**

### CODE REVIEW

File: server/graphql/v2/mutation/VirtualCardMutations.ts [hunk 1]
```diff
@@ -58,9 +59,22 @@ const virtualCardMutations = {
...
+      await models.Activity.create({
+        type: activities.COLLECTIVE_VIRTUAL_CARD_ASSIGNED,
+        UserId: req.remoteUser.id,
+        CollectiveId: collective.id,
+        data: {
+          assignee: assignee.activity,
+          collective: collective.activity,
+          host: host.activity,
+        },
+      }).catch(e => logger.error('An error happened when creating the COLLECTIVE_VIRTUAL_CARD_ASSIGNED activity', e));
```
Comment (SudharakaP) [inline server/graphql/v2/mutation/VirtualCardMutations.ts:75]:
```suggestion
      }).catch(e => logger.error(`An error occured when creating the ${activities.COLLECTIVE_VIRTUAL_CARD_ASSIGNED} activity`, e));
```

**Review by SudharakaP — APPROVED** [review]

File: templates/emails/collective.virtualcard.assigned.hbs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+{{> header}}
+
+<center>
+  <h1>
+    {{host.name}} assigned a new Virtual Card to {{collective.name}}
+  </h1>
+</center>
+
+<p>
+  {{host.name}} assigned a new Virtual Card to your collective {{collective.name}}. This means that you also agreed with
```
Comment (piamancini) [inline templates/emails/collective.virtualcard.assigned.hbs:13]: {{host.name}} assigned a new Virtual Card to your collective {{collective.name}}. As a reminder, you have agreed to their virtual card use policy and you can find it here {{LINK}}

File: templates/emails/collective.virtualcard.assigned.hbs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  the policy defined by {{host.name}} for using Virtual Cards.
+</p>
+<p>
+  <strong>Please, make sure to keep all expenses up-to-date, uploading the payment receipt to every charge you have on
```
Comment (piamancini) [inline templates/emails/collective.virtualcard.assigned.hbs:17]: <strong>Please, make sure to keep all expenses up-to-date, uploading the payment receipt to every charge you have on this card.</strong> After each charge, you will receive an email with an invitation to submit the receipt of the purchase made. Your host may pause the card for future charges until you do.

Comment (piamancini) [reply templates/emails/collective.virtualcard.assigned.hbs:17]: For more info, please go ahead and read the virtual cards FAQs {{LINK}}

**Pull request merged**